docs/UserGuide.md

```diff
@@ -61,139 +77,130 @@ Adds a new class to the list of classes.
-* Class must have been created to add a student inside that class
 
-**Example**:
-`add /s John /c cs2103t`
+**Examples**:
+* `add /s John /c cs2103t`
 
 **Successful Output**:
-"STUDENT_NAME has been added to CS2103T"
+* STUDENT_NAME has been added to CLASS_NAME
 
 **Unsuccessful Output**:
-* "Class name was not specified"
-* "CLASS_NAME does not exist"
-
+* Class name not specified
+* CLASS_NAME does not exist
 
 ### Removing a student : `remove`
 
 Removes an existing student from a class.
 
 **Command Format**: `remove /s STUDENT_NAME /c CLASS_NAME`
-* STUDENT_NAME and CLASS_NAME are compulsory parameters
-* CLASS_NAME is case-insensitive
-* STUDENT_NAME only accepts Alphabets
-* Student must have been added into the class to be removed
 
-* CLASS_NAME must be unique (cannot have two classes with the same name)
-
-**Example:**
-Remove /s John /c CS2103T
-
-**Successful Output:**
-“STUDENT_NAME has been removed from CLASS_NAME”
-
-**Unsuccessful Output:**
-“Class name not specified”
-“STUDENT_NAME is not found in CLASS_NAME”
-“CLASS_NAME does not exist”
-
-### Viewing a class : `view`
-Prints out the detailed information (class name, class schedule, enrolled student list) about a specific class.
-**Command Format**: `view /c CLASS_NAME`
 * CLASS_NAME is not case-sensitive
-* Class must have been created to be viewed
+* Class and student must already exist
+* STUDENT_NAME only accepts alphabetical characters
 
-**Examples**:
-view CS2103T
-view cs2103t
+**Examples:**
+* Remove /s John /c CS2103T
 
 **Successful Output:**
-“CLASS_NAME”:
-Enrolled students: (...)
+* STUDENT_NAME has been removed from CLASS_NAME
 
 **Unsuccessful Output:**
-“CLASS_NAME” does not exist
+* Class name not specified
+* CLASS_NAME does not exist
+* STUDENT_NAME is not found in CLASS_NAME
 
 ### Exiting the program : `exit`
 
 Exits the program.
 
-Format: `exit`
+**Command Format:** `exit`
 
 ### Saving the data
 
-EduTrack data are saved in the hard disk automatically after any command that changes the data. There is no need to save manually.
+EduTrack's data are saved in the hard disk automatically after any command that changes the data. There is no need to save manually.
 
-### Adding a list of students
+### Adding a list of students : `coming in v1.3`
 [Coming soon...]
 
-### Updating a Class
+### Updating a class : `coming in v1.3`
 [Coming Soon...]
 
-### Updating a Student Profile
+### Updating a class note : `coming in v1.3`
 [Coming Soon...]
 
-### Taking attendance for a lesson
+### Updating a student : `coming in v1.3`
 [Coming Soon...]
 
-### Updating a Class note
+### Adding a lesson : `coming in v1.3`
 [Coming Soon...]
 
-### Adding a lesson to a Class Schedule
+### Removing a lesson : `coming in v1.3`
 [Coming Soon...]
 
-### Removing a lesson from a Class Schedule
+### Taking attendance : `coming in v1.3`
 [Coming Soon...]
 
-### Finding the profile of a Student from a Class
+### Finding a student : `coming in v1.3`
```

Comment:
Thank you so much for helping to standardise, it must have been tiring to thoroughly scan through the whole user guide to pick out for these inconsistencies!!